docs/docs/persistent-volumes.md

```diff
@@ -39,69 +43,77 @@ Additionally, you need to set a `residency` in order to tell Marathon to setup a
+Mesos creates persistent volumes to hold your application's stateful data. Because persistent volumes are local to an agent, the stateful task using this data will be pinned to the agent it was initially launched on, and will be relaunched on this node whenever needed. You do not need to specify any constraints for this to work: when Marathon needs to launch a task, it will accept a matching Mesos offer, dynamically reserve the resources required for the task, create persistent volumes, and make sure the task is always restarted using these reserved resources so that it can access the existing data.
 
-- the application is deleted (in which case volumes of all its tasks are destroyed, and all reservations are deleted)
-- you explicitly delete one or more suspended tasks with a `wipe=true` flag
+Once a task that used persistent volumes has terminated, its metadata will be kept. This metadata will be used to launch a replacement task when needed.
+
+For example, if you scale down from 5 to 3 instances, you will see 2 tasks in the `Waiting` state along with the information about the persistent volumes the tasks were using as well as about the agents on which they are placed. Marathon will not unreserve those resources and will not destroy the volumes. When you scale up again, Marathon will attempt to launch tasks that use those existing reservations and volumes as soon as it gets a Mesos offer containing the labeled resources. Marathon will only schedule unreserve/destroy operations when:
+
+- the application is deleted (in which case volumes of all its tasks are destroyed, and all reservations are deleted).
+- you explicitly delete one or more suspended tasks with a `wipe=true` flag.
 
 If reserving resources or creating persistent volumes fails, the created task will timeout after the configured `task_reservation_timeout` (default: 20 seconds) and a new reservation attempt will be made. In case a task is `LOST` (because its agent is disconnected or crashed), the reservations and volumes will not timeout and you need to manually delete and wipe the task in order to let Marathon launch a new one.
 
 ## Potential Pitfalls
 
-You should be aware of the following issues and limitations when using stateful applications in Marathon that make use of dynamic resevations and persistent volumes.
+Be aware of the following issues and limitations when using stateful applications in Marathon that make use of dynamic resevations and persistent volumes.
 
 ### Static Reservations
```

Comment:
Yes and no. It's about not being able to use static reservations for stateful services. Can you think of a better title?